### ISSUE TITLE
Accessibility Checker Panel Status language constants do not cater for multiple plural forms

### ISSUE THREAD

**Issue by dyfrig** [joomla-projects/joomla-a11y-checker#66]
The system plugin Joomla Accessibility Checker needs to be changed so that the following language constants cater for languages with multiple plurals

PLG_SYSTEM_JOOA11Y_PANEL_STATUS_2="1 accessibility error and %(warningCount) warnings found."
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_3="%(errorCount) accessibility errors and 1 warning found."
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_4="%(errorCount) accessibility errors and %(warningCount) warnings found."
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_6="%(errorCount) accessibility errors found."
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_8="Please review %(warningCount) warnings."
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_10="%(warningCount) <span class="jooa11y-visually-hidden">warnings found."
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_11="%(totalCount) <span class="jooa11y-visually-hidden">total issues found."

It has also been pointed out that constants should not end with digits when they are not plurals so I would suggest rewording the constants and language strings to something like which would avoid both issues.

;Panel status
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_1_MESSAGE="Accessibility errors: 1, Warnings: 1"
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_10_MESSAGE="<span class=\"jooa11y-visually-hidden\">Warnings found:</span> %(warningCount)"
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_11_MESSAGE="<span class=\"jooa11y-visually-hidden\">total issues found:</span> %(totalCount)"
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_2_MESSAGE="Accessibility errors: 1, Warnings: %(warningCount)"
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_3_MESSAGE="Accessibility errors: %(errorCount), Warnings: 1"
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_4_MESSAGE="Accessibility errors: %(errorCount), Warnings: %(warningCount)"
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_5_MESSAGE="Accessibility errors: 1"
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_6_MESSAGE="Accessibility errors: %(errorCount)"
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_7_MESSAGE="Please review warning."
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_8_MESSAGE="Please review warnings [%(warningCount)]."
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_9_MESSAGE="No accessibility errors found."
PLG_SYSTEM_JOOA11Y_PANEL_STATUS_12_MESSAGE="The item you are trying to view is not visible; it may be hidden or inside of an accordion or tab component. Here's a preview="

This has been discussed on https://github.com/joomla/joomla-cms/issues/37053 and also on Translation Forum at https://forum.joomla.org/viewtopic.php?f=54&t=991925

**Comment by brianteeman** [comment]
With this proposed change would we not be able to remove status2 and status3 and just use status4 etc?
At least in English it is valid to say "Apples: 1" "Apples: 2" "Apples:3"
Would that work for Welsh as well?

**Comment by dyfrig** [comment]
@brianteeman Yes, it would be fine for Welsh.

**Comment by brianteeman** [comment]
great - will put it into the next release

**Comment by brianteeman** [comment]
Closed as there is a pr for this now and i will be in the next release

**Comment by wojsmol** [comment]
During testing of Polish language for Joonla 4.1 I discovered that HTML entities liker `&nbsp;'` are not parsed in above language constants.

**Comment by brianteeman** [comment]
There aren't any html entities

**Comment by wojsmol** [comment]
@bembelimen In original string not but in Polish we wont to use `&nbsp;` in few places to avoid breaking lines before `i` the equivalent of `and`.  You can see that here https://github.com/JoomlaPolska/jezyk-J4/blob/v4.1.0/administrator/language/pl-PL/plg_system_jooa11y.ini#L52-L64.

**Comment by brianteeman** [comment]
This is not a problem with jooa11y. I strongly suspect it is with the way joomla includes strings with JavaScript. As such please open an issue on the main joomla tepo
